python/eos/parameter.py

```diff
@@ -128,3 +128,10 @@ def FromWCxf(wc):
         return parameters
 
 
+    @staticmethod
+    def assert_valid_name(name):
+        "Raise error if 'name' is not a valid EOS paramter name"
+
+        valid_names = [p.name() for p in Parameters()]
```

Comment:
That seems dirty. Why don't you add a ``Parameters::has(const std::string & key)`` method and export it?

Reply:
Done!